release-notes/v1.2.0-rc.1.yaml

```diff
@@ -0,0 +1,179 @@
+      Fixed HTTPHeaderFilter processing doesn't correctly support multiple header values
+      Added Support for session persistence in HTTPRoute rules
+      Added Support for the Backend resource for ExtAuth
+      Fixed multiple reference grants in same namespace
+      Fixed upstream get unwanted /.
+      Added statPrefix for HCM and TCPProxy
+      Fixed creation of SecurityPolicy with targetSelectors fails
+      Fixed wrong gateway is chosen as HTTPRoute parent
+      Fixed override issue for EEP
+      Fixed nil pointer err translating hash load balancing
+      Fixed ratelimit does not work across multiple GatewayClasses
+      Fixed upstream mTLS only works for HTTPS listeners
+      Fixed nil pointer if backedtls.minVersion is set but backedtls.maxVersion is not
+      Added Support for target selectors on Envoy Gateway Extension Server policies
+      Added Support for non-Kubernetes Backends for TLSRoute
+      Fixed empty connection limit causes XDS rejection
+      Fixed ratelimit not working with both headers and cidr matches
+      Added Support for fallback to the Backend API
+      Added Support for ipv4/ipv6 dual stack in the listeners and clusters
+
+  - area: providers
+    change: |
+      Added file resource provider for EG standalone mode
+      Fixed EDS didn't update when deployments was created after services
+      Fixed repeated resources and optimize memory usage
+      Added Support for host-infra provider(experimental)
+      Added Support for reloadable EnvoyGateway configuration
+
+  - area: infrastructure
+    change: |
+      Fixed RBAC issue for deleting infrastructure resources
+      Fixed customized infrastructure resources not being deleted
+      Added Support for adding Labels to the Envoy Service
+      Fixed Gateways never become ready/programmed when running Envoy as a Daemonset
+      Fixed Ratelimit Deployment ignoring pod labels and annotation merge
+      Fixed the API Server receives unnecessary requests
+      Updated shutdown settings for the Envoy Proxy: default minDrainDuration is now 10s, default drainTimeout is now 60s, and terminationGracePeriodSeconds is now 360s
+      Fixed terminating envoy pods don't respond with "Connection: close" (H1) or GOAWAY(H2) on shutdown, switch to an immediate drain strategy
+      Added Support for custom name for ratelimit deployment
+      Fixed ratelimit statsd not working
+      Fixed not generating selector of deployment/daemonset based on the custom label configuration of EnvoyProxy
+      Reduced readinessProbe failureThreshold and periodSeconds
+      Added default SecurityContext for EG components
+      Added startupProbe to all provisioned containers
+
+  - area: xds
+    change: |
+      Bumped go-control-plane to v0.13.1
+
+  - area: cli
+    change: |
+      Added Support for local validations for egctl translate and file provider
+      Fixed egctl experimental translate with error ns
```

Comment:
not sure what this means, can we add some more clarity like - `fixed an issue in egctl x translate where...`